packages/google_adsense/lib/src/js_interop/adsbygoogle.dart

```diff
@@ -0,0 +1,30 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:js_interop';
+
+/// Binding to the `adsbygoogle` JS global.
+///
+/// See: https://support.google.com/adsense/answer/9274516?hl=en&ref_topic=28893&sjid=11495822575537499409-EU
+@JS('adsbygoogle')
+external AdsByGoogle get adsbygoogle;
+
+/// The Dart definition of the `adsbygoogle` global.
+@JS()
+@staticInterop
+abstract class AdsByGoogle {}
```

Comment:
This style of JS-interop is a little bit outdated, this is now done with "Extension Types".

As an example, see how the "console" type (and object) is defined in `package:web`

* https://github.com/dart-lang/web/blob/d3a61aa9b4e43cdaf949a69c159038cdecf5681a/web/lib/src/dom/console.dart